src/components/view/Navigation/index.js

```diff
@@ -41,7 +41,7 @@ class Navigation extends React.Component {
     }
 
     handleToggle () {
-        // doesn't show nevigation unless logged in.
+        // doesn't show nvigation unless logged in.
```

Comment:
Spelleng!

Reply:
LOL...  sorry for the original misspelling.